business/checkers/destinationrules/disabled_namespacewide_mtls_checker.go

```diff
@@ -19,34 +19,34 @@ func (m DisabledNamespaceWideMTLSChecker) Check() ([]*models.IstioCheck, bool) {
 		return validations, true
 	}
 
-	// otherwise, check among Policies for a rule enabling mTLS
-	for _, mp := range m.MTLSDetails.Policies {
-		if enabled, mode := kubernetes.PolicyHasMTLSEnabled(mp); enabled {
-			// If Policy has mTLS enabled in STRICT mode
+	// otherwise, check among PeerAuthentications for a rule enabling mTLS
+	for _, mp := range m.MTLSDetails.PeerAuthentications {
+		if enabled, mode := kubernetes.PeerAuthnHasMTLSEnabled(mp); enabled {
+			// If PeerAuthn has mTLS enabled in STRICT mode
 			// traffic going through DestinationRule won't work
 			if mode != "PERMISSIVE" {
 				check := models.Build("destinationrules.mtls.policymtlsenabled", "spec/trafficPolicy/tls/mode")
 				return append(validations, &check), false
 			} else {
-				// If Policy has mTLS enabled in PERMISSIVE mode
+				// If PeerAuthn has mTLS enabled in PERMISSIVE mode
 				// traffic going through DestinationRule will work
-				// no need for further analysis in MeshPolicies
+				// no need for further analysis in MeshPeerAuthentications
 				return validations, true
 			}
 		}
 	}
 
-	// In case any Policy enables mTLS, check among MeshPolicies for a rule enabling it
-	// ServiceMeshPolicies are a clone of MeshPolicies but used in Maistra scenarios
-	// MeshPolicies and ServiceMeshPolicies won't co-exist, only ony array will be populated
-	mPolicies := m.MTLSDetails.MeshPolicies
+	// In case any PeerAuthn enables mTLS, check among MeshPeerAuthentications for a rule enabling it
+	// ServiceMeshPolicies are a clone of MeshPeerAuthentications but used in Maistra scenarios
```

Comment:
Yes, probably Maistra will follow the same pattern here, but yes, to be defined when they start to migrate these objects as well.